src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,9 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got " + number);
+        }
+        return Math.pow(number,0.5);
```

Comment:
```suggestion
        return Math.sqrt(number);
```
your solution is good, but it is more logical to use Math.sqrt